src/librawspeed/decoders/DngDecoder.h

```diff
@@ -52,7 +52,8 @@ class DngDecoder :
 class DngStrip {
 public:
   DngStrip() { h = offset = count = offsetY = 0;};
-  ~DngStrip() {};
+  ~DngStrip() = default;
+  ;
```

Comment:
same here (Now I see how this came to be: there was an unnecessary ';' in the first place...)